config/config.sample.php

```diff
@@ -1387,6 +1387,17 @@
  */
 'filesystem_check_changes' => 0,
 
+/**
+ * This config value avoids infinite loops for seldom cases where a file renaming
```

Comment:
```suggestion
 * Define unsuccessful mountpoint rename attempts
 * This config value avoids infinite loops for seldom cases where a file renaming
```
@VicDeo There was a "headline" missing. This headline will be automatically used as item text in the table of contents on top of the document created by the config-to-docs run. I forgot to mention this when making the first text proposal. This commit suggestion fixes it. If you agree, you can securly commit it.